catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/AbstractComponentTest.java

```diff
@@ -47,6 +48,8 @@
         bundlesToStart().stream().map(this::startBundle).toArray(Option[]::new);
 
     return options(
+        editConfigurationFilePut(
+            "etc/org.apache.karaf.features.cfg", "serviceRequirements", "disable"),
```

Comment:
The effect of setting it is that Karaf will not fail to resolve a bundle if there is no other bundle in the context that provides that capability. I forget the exact bundles this started failing on, but there were multiples. Since we are disabling this in production, I thought it made sense to do the same in our itests. https://github.com/codice/ddf/blob/master/distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg#L70
To @emmberk's point, I agree it should go in one of the other `configuration` methods. I had just thrown this in here.